.github/workflows/ondemand.yml

```diff
@@ -0,0 +1,45 @@
+name: Regression on demand tests
+
+on:
+  workflow_dispatch:
```

Comment:
Have you tested it?

Reply:
@katspaugh no, from what I understand it must be merged to dev first so the workflow is available